ddd-koans/Entity/Lifecycle/task.html

```diff
@@ -23,5 +23,17 @@ <h2>Lifecycle</h2>
     To sum up, a bar can not yet exist, be in an incomplete state or be in a complete state.
     The lifecycle must be modelled in the code then.
 </p>
-<h2>Examples</h2>
+<h2>Task</h2>
+<p>
+    To begin with the lifecycle, please implement <code>Bar#state()</code> method which returns
+    Bar.State.Complete for a complete bar, Bar.State.Incomplete for an incomplete bar and throws
+    BarWithTooManyNotesException when there are too many notes in the bar.
+</p>
+<h2>Calculation examples</h2>
+<h3>Incomplete</h3>
+<!--TODO: Image with bar which is imcoplete and with calculations-->
+<h3>Complete</h3>
+<!--TODO: Image with bar which is complete and with calculations-->
+<h3>Invalid</h3>
+<!--TODO: Image with bar which is invalid and with calculations-->
```

Comment:
should I prepare some images?

Reply:
You can, however, I can also do it. It was just too much for me when I was creating the whole PR.